eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Pool.hs

```diff
@@ -211,6 +217,11 @@ poolCertTransition (PoolEnv slot pp) ps@PState {psStakePoolParams, psFutureStake
             { psFutureStakePoolParams = eval (psFutureStakePoolParams ⨃ singleton ppId poolParams)
             , psRetiring = eval (setSingleton ppId ⋪ psRetiring)
             }
+  -- FOR CONWAY
+  --
+  --   POOL-retirepool : let open PoolParams poolParams in
+  --     pp ⊢ ⟦ pools , retiring ⟧ᵖ ⇀⦇ retirepool c e ,POOL⦈
+  --          ⟦ pools , ❴ c , e ❵ᵐ ∪ᵐˡ retiring ⟧ᵖ
```

Comment:
This should also not change between Babbage and Conway.

Reply:
Noted! Do you want me to take this out or keep it though?